llama_index/llms/vertex.py

```diff
@@ -128,7 +128,6 @@ def class_name(cls) -> str:
     @property
     def metadata(self) -> LLMMetadata:
         return LLMMetadata(
-            is_chat_model=True,
```

Comment:
I think probably, we need to either let the user set this in the constructor, or develop some map of completion vs. Text models. Probably better to do the former

Reply:
Actually. Looks like we already know chat vs non-chat in the constructor